### ISSUE TITLE
User-configurable UI for Radicle commands in Source Control View

### ISSUE THREAD

**Issue by maninak** [cytechmobile/radicle-vscode-extension#24]
Under a setting group `Source Control View` have the following boolean settings:

1. `Radicle submenu in three-dot menu` (default: `true`)
2. `Radicle menu next to title` (default: `false`)
3. `Radicle command buttons next to title` (default: `false`)
3.1. We should note in the description that individual buttons are individually hide-able with r-click (native VS Code feature)
    
    https://user-images.githubusercontent.com/7542054/225389526-6cf214e7-cec3-499f-b43e-0d73ef5f4465.mp4

>Example UI of such a setting in VS Code ("VersionLens" is another extension's name)
>
>![image](https://user-images.githubusercontent.com/7542054/225387267-2be5bfa8-cf5a-4a7e-9caf-1cfd9c8ac872.png)

The above assumes that we have extension settings implemented.

Depends on #8.

**Comment by gsaslis** [comment]
@maninak I'm moving this to the backlog for now. I see this as a nice-to-have that can wait for a version or two.
